pkgs/by-name/pl/plakar/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  stdenv,
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  fuse,
+}:
+buildGoModule (finalAttrs: {
+  pname = "plakar";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "PlakarKorp";
+    repo = "plakar";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-4fJB00poA06aQC//zw6L0bRXa8KauFJfZbniCu+vrac=";
+  };
+
+  vendorHash = "sha256-UxTETtR7LNOavIS2oJRVKx/Ns2bvzJe+Enh8xFhM2mI=";
+
+  buildInputs = [
+    fuse
+  ];
+
+  checkFlags =
+    let
+      skippedTests = [
+        # mount: fusermount: exec: "fusermount": executable file not found in $PATH
+        "TestExecuteCmdMountDefault"
+      ]
+      ++ lib.optionals stdenv.isDarwin [
+        "TestBTreeScanMemory"
+        "TestBTreeScanPebble"
+      ];
+    in
+    [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
+
```

Comment:
In my local tests with version 1.0.3 of plakar all tests succeeded and none need to be skipped. Can someone verify my claim please :)

```suggestion
```